bluewin.ch

```diff
@@ -1,41 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
-  <emailProvider id="bluewin.ch">
+  <emailProvider id="Swisscom">
     <domain>bluewin.ch</domain>
-    <!-- See also bluemail.ch -->
-    <displayName>bluewin.ch</displayName>
-    <displayShortName>bluewin.ch</displayShortName>
+    <domain>bluemail.ch</domain>
+    <displayName>Bluewin Mail</displayName>
+    <displayShortName>Bluewin</displayShortName>
     <incomingServer type="imap">
       <hostname>imaps.bluewin.ch</hostname>
       <port>993</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="pop3">
       <hostname>pop3s.bluewin.ch</hostname>
       <port>995</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
+      <username>%EMAILADDRESS%</username>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <outgoingServer type="smtp">
       <hostname>smtpauths.bluewin.ch</hostname>
       <port>465</port>
       <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-encrypted</authentication>
+      <username>%EMAILADDRESS%</username>
+      <authentication>password-cleartext</authentication>
     </outgoingServer>
-    <documentation url="http://smtphelp.bluewin.ch/swisscomdtg/setup/"/>
+    <documentation url="http://smtphelp.bluewin.ch/" />
   </emailProvider>
   <webMail>
-    <loginPage url="https://rich.sso.bluewin.ch/cp/applink/sso/Login?d=%EMAILDOMAIN%" />
-    <loginPageInfo
-      url="https://rich.sso.bluewin.ch/cp/applink/sso/Login?d=%EMAILDOMAIN%">
-      <username>%EMAILLOCALPART%</username>
-      <usernameField id="username" name="user" />
-      <passwordField name="password" />
-      <loginButton name="anmelden" />
-    </loginPageInfo>
+    <loginPage url="https://webmail.bluewin.ch/" />
```

Comment:
Can the user no longer log in even with their email address and password?

Please fix the values rather than removing them.

Reply:
FWIW, neither of these URLs, not even the documentation page, respond to me when I try to open them in a browser.

Reply:
The URLs are tested and correct, they redirect to the correct pages and are the communicated entry points.
The login to webmail works via a different login which is used to manage your whole account, potentially including multiple mail addresses. This login is unrelated to the available mail addresses and can not be systematically correlated.

Reply:
FWIW, the URLs now work for me. They did not 2 hours ago.

Reply:
Interesting, these URLs haven't changed in a while, they just were never updated in the ISPDB

Reply:
So, is there no way you can maintain the ability of user to log in with their email address and password to see their webmail?
If there is, could you please fix the form values?
If not, I would be interested to know why not.

Reply:
Simply put: there is currently no way to login to Webmail with your email address and password, there hasn't been in years and it probably wont be possible in the foreseeable future.
It has all to do with Single Sign On, corporate decisions and account modelling but to go into the details here would probably help no one.